paramiko/ssh_exception.py

```diff
@@ -67,9 +67,7 @@ def __init__(self, explanation, types):
         self.allowed_types = types
 
     def __str__(self):
-        return "{}; allowed types: {!r}".format(
-            self.explanation, self.allowed_types
-        )
+        return "{}; allowed types: {!r}".format(self.explanation, self.allowed_types)
```

Comment:
Why changed? It is not needed for adding log entry. Also the line is exceeded 79 characters.

```
./paramiko/ssh_exception.py:70:80: E501 line too long (85 > 79 characters)
```

Reply:
I have no idea what happened here. I’ll fix it